docs/definitionOfdone.md

```diff
@@ -0,0 +1,36 @@
+# Definition of done
```

Comment:
How are you seeing this document implemented? Are these new status points that need to get checked off on all tickets (where they apply)? Is it just something for reference should a question about our scope of work come up?

Reply:
That is a great question. I'd love for this to be something that is not easily ignored. Maybe something like this
https://victoria.dev/blog/an-automatic-interactive-pre-commit-checklist-in-the-style-of-infomercials/

Reply:
See https://github.com/AlaskaAirlines/WC-Generator/pull/120/commits/a9ef123ca04118b972139891dcfe4837a2f583cf